kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/KyuubiDatabaseMetaData.java

```diff
@@ -60,11 +60,10 @@ public ResultSet getTables(String catalog, String schemaPattern,
         if (tStatus.getStatusCode() != TStatusCode.SUCCESS_STATUS) {
             throw new HiveSQLException(tStatus);
         }
-        new HiveQueryResultSet.Builder(conn)
+        return new HiveQueryResultSet.Builder(conn)
                 .setClient(client)
                 .setSessionHandle(sessHandle)
                 .setStmtHandle(getTableResp.getOperationHandle())
                 .build();
-        return super.getTables(catalog, schemaPattern, tableNamePattern, types);
```

Comment:
It makes the method make no sense, just invokes the superclass.

Reply:
@yaooqinn FYI

Reply:
Iet's update in the PR desc then

Reply:
Updated.